src/Compilers/CSharp/Portable/Utilities/ValueSetFactory.NumericValueSet.cs

```diff
@@ -19,23 +19,24 @@ internal static partial class ValueSetFactory
         /// <summary>
         /// The implementation of a value set for an numeric type <typeparamref name="T"/>.
         /// </summary>
-        private sealed class NumericValueSet<T, TTC> : IValueSet<T> where TTC : struct, INumericTC<T>
+        private sealed class NumericValueSet<T, TTC> : IValueSet<T> where TTC : class, INumericTC<T>
```

Comment:
Do we really need this type parameter? It looks like we can simply use ```INumericTC<T>``` everywhere where it is used.

Reply:
This comment was about ```TTC```